scripts/generate_header.py

```diff
@@ -155,6 +155,21 @@ class DeviceBase:
 
 # sub templates
 # generate sub parts of template
+
+def integer_from_python_to_c(an_int: int) -> str:
+    """
+    Takes a positive python integer and return a string that can
+    be used as a constant in C. This means that a L will be prepended
+    if the value is greater than 32 bits.
```

Comment:
If you do decide to keep this function, then can you make the following changes to the docstring?

- The word "constant" should be replaced with "literal"
- The word "prepended" should be replaced with "appended"
- "L" should be replaced with "LL", since `long` is a different type than `long long` in C and we should make sure we are precise here.

Reply:
fixed. always appends LL, if greater than/equal to zero, appends ULL

Yes, I know zero is zero regardless of sign, but it felt better for some reason to call zero unsigned.